exercises/practice/armstrong-numbers/.meta/lib/example.dart

```diff
@@ -3,17 +3,16 @@ import 'dart:math' show pow;
 class ArmstrongNumbers {
```

Comment:
The import at the top of this file can be removed
```diff
-    import 'dart:math' show pow;
```

Reply:
Good catch, thanks!